resources/webroot/index.js.mustache

```diff
@@ -2446,7 +2446,7 @@ function getPokemonPopupContent (pokemon) {
         }
         let infoString = '#' + ranking.rank + ' (' + Math.round(ranking.percentage * 1000) / 10 + '%)'
         if (ranking.cp !== null) {
-          infoString += ' @' + ranking.cp + 'CP (Lvl. ' + (ranking.level) + '/' + (ranking.cap) + ')'
+          infoString += ' @' + ranking.cp + i18n('map_pokemon_pvp_pc')+' ('+i18n('map_pokemon_level')+ ' ' + (ranking.level) + '/' + (ranking.cap) + ')'
```

Comment:
`map_pokemon_pvp_pc`, misspelling?

Reply:
Not quite, more like a brain fart. PC is in Spanish (Puntos de Combate). I'll change it!